src/simulators/matrix_product_state/svd.cpp

```diff
@@ -28,9 +28,10 @@
 #include "framework/utils.hpp"
 #include "framework/linalg/almost_equal.hpp"
 
+// default values
 #define mul_factor 1e2
 #define tiny_factor 1e30
-#define THRESHOLD 1e-9
+#define THRESHOLD 1e-9 // threshold for re-normalization after approximation
```

Comment:
Can these be made const vars or configurable vars rather than compile time defines?